apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -5495,6 +5495,9 @@
       "title": "Impossible to calculate amount and fees",
       "description": "Impossible to calculate amount and fees"
     },
+    "InvalidRecipientForTokenTransfer": {
```

Comment:
should we do the same translation for Ledger Mobile?

Reply:
Yes definitely, I am sorry it wasn't there but now I have added it.